### PR TITLE
[MiniTwoScan] Added Website Support

### PR DESCRIPTION
**Pull request by M4lho**

### CODE REVIEW

File: src/web/mjs/connectors/MiniTwoScan.mjs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import WordPressMadara from './templates/WordPressMadara.mjs';
+
+export default class minitwoscan extends WordPressMadara {
```
Comment (27136773) [inline src/web/mjs/connectors/MiniTwoScan.mjs]: Use pascal casing for class name (same as filename)

**Comment by 27136773** [comment]
Is the website not providing an icon?
Check if they have facebook/twitter and get icon from there.
Or use a copy of the default item (e.g. from antisensescan)

File: src/web/mjs/connectors/MiniTwoScan.mjs (hunk 1 above)
Comment (M4lho) [reply src/web/mjs/connectors/MiniTwoScan.mjs]: Okay, updated.

**Comment by M4lho** [comment]
> Is the website not providing an icon?
> Check if they have facebook/twitter and get icon from there.
> Or use a copy of the default item (e.g. from antisensescan)

they have a favicon but consider it to be too small and of poor quality.
minitwo

Their Facebook page image is a generic from some manga.

**Comment by M4lho** [comment]
Added ICO.

**Review by 27136773 — APPROVED** [review]
Looks good

**Pull request merged**